common/models/model-method.js

```diff
@@ -10,4 +12,53 @@ module.exports = function(ModelMethod) {
    * @class ModelMethod
    * @inherits WorkspaceEntity
    */
+  app.once('ready', function() {
+    ready(ModelMethod);
+  });
 };
+
+
+function ready(ModelMethod) {
+  var debug = require('debug')('workspace:model-method');
+  var PackageDefinition = app.models.PackageDefinition;
+  var ModelDefinition = app.models.ModelDefinition;
+  
+  PackageDefinition.observe('after save', function getPackageDef(ctx, next) {
```

Comment:
I find this suspicious. What if `PackageDefinition` is never saved and only loaded from fs?